### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by ajitbohra**
- readme.txt plugin download link updated (Current link returns 404)
- readme.md created for visually appealing information for users

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+>**Note:** If you think code can be improved or have any suggestion/issue feel free to send a PR or [open an issue](https://github.com/Instamojo/Woo-Instamojo/issues).
+
+
+Installation
+------------
+
+This section describes how to install the plugin and get it working.
+
+1. Make sure you've installed the [WooCommerce plugin](https://wordpress.org/plugins/woocommerce/) and it is activated. Without WooCommerce this plugin won't work.
+2. Search for "Instamojo Payment Gateway for WooCommerce" on the WordPress Plugin directory or [download it](http://downloads.wordpress.org/plugin/woocommerce-instamojo.zip).
```
Comment (ashwch) [inline readme.md]: Let's fix the URL here  as well.

**Review by ashwch — CHANGES_REQUESTED** [review]
Can we rename the file to README.md?

File: readme.md (hunk 1 above)
Comment (ajitbohra) [reply readme.md]: Plugin URL has been updated

**Comment by ajitbohra** [comment]
@ashwch as suggested "readme.md" is renamed to "README.md"

**Comment by ashwch** [comment]
@ajitbohra Thanks. Let's fix the spelling in the banner as well: "competative".

**Comment by ajitbohra** [comment]
@ashwch check PR #7 banner typo is fixed as well  :)

**Review by ashwch — APPROVED** [review]
Thanks!

**Pull request merged**